src/style.css

```diff
@@ -1,3 +1,31 @@
 body {
-  background-color: bisque;
+  background-color: #f3e5e5;
+}
+
+header {
+  background-image: url(../Assets/logo.jpg);
```

Comment:
- [ ] Please rename the `Assets` folder to assets and change the reference here because the trend is to name sub folders in small letters.